packages/client/src/containers/LandingPage/Dashboard.jsx

```diff
@@ -0,0 +1,178 @@
+    try {
+      const response = await fetch(`${apiURL()}/teamMembers/${editMemberId}`, {
+        method: 'PATCH',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          first_name: editMemberFirstName,
+          last_name: editMemberLastName,
+        }),
+      });
+      if (!response.ok) {
+        throw new Error('Failed to edit team member');
+      }
+      fetchTeamMembers();
+      setEditMemberId(null);
+      setEditMemberFirstName('');
+      setEditMemberLastName('');
+    } catch (error) {
+      console.error(error); // Changed to log the error without specific message
+    }
+  };
+
+  const handleDeleteMember = async (id) => {
+    const confirmDelete = window.confirm(
+      'Are you sure you want to delete this team member?',
+    );
+    if (confirmDelete) {
+      try {
+        const response = await fetch(`${apiURL()}/teamMembers/${id}`, {
+          method: 'DELETE',
+        });
+        if (!response.ok) {
+          throw new Error('Failed to delete team member');
+        }
+        setTeamMembers((prevMembers) =>
+          prevMembers.filter((member) => member.member_id !== id),
+        );
+      } catch (error) {
+        console.error(error); // Changed to log the error without specific message
+      }
+    }
+  };
+
+  const handleAddMember = async () => {
+    try {
+      const response = await fetch(`${apiURL()}/teamMembers/addMember`, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+          first_name: newMemberFirstName,
+          last_name: newMemberLastName,
+        }),
+      });
+      if (!response.ok) {
+        throw new Error('Failed to add team member');
+      }
+      fetchTeamMembers();
+      setShowAddModal(false);
+    } catch (error) {
+      console.error(error); // Changed to log the error without specific message
+    }
+  };
+
+  return (
+    <div>
+      <h1>Dashboard</h1>
+      <div>
+        <h2>Team Members</h2>
+        <ul>
+          {teamMembers.map((member) => (
+            <li key={member.member_id}>
+              {member.first_name} {member.last_name}
+              <button
+                type="button"
+                onClick={() => {
+                  setEditMemberId(member.member_id);
+                  setEditMemberFirstName(member.first_name);
+                  setEditMemberLastName(member.last_name);
+                }}
+              >
+                Edit
+              </button>
+              <button
+                type="button"
+                onClick={() => handleDeleteMember(member.member_id)}
+              >
+                Delete
+              </button>
+            </li>
+          ))}
+        </ul>
+      </div>
+      <button type="button" onClick={() => setShowAddModal(true)}>
+        Add Team Member
+      </button>
+      {showAddModal && (
+        <div>
```

Comment:
This whole add modal could be in a different component so it becomes more readable